setup.py

```diff
@@ -67,8 +67,8 @@
         "Programming Language :: Python :: 3.11",
         "Framework :: Django",
         "Framework :: Django :: 3.2",
-        "Framework :: Django :: 4.0",
```

Comment:
As Django 4.0 wasn't tested in the tox matrix and it's no longer a supported version I have removed it here.